class-post-public.php

```diff
@@ -1221,20 +1223,11 @@ public function get_post_translations( $post ) {
 		foreach ( $langs as $lang )
 			$post_types[] = bbl_get_post_type_in_lang( $post->post_type, $lang->code );
 
-		// Get all the translations in one cached DB query
-		$args = array(
-			// We want a clean listing, without any particular language
-			'bbl_translate' => false,
-			'include' => $post_ids,
-			'post_type' => $post_types,
-			'post_status' => array( 'publish', 'pending', 'draft', 'future' ),
-		);
-		$posts = get_posts( $args );
```

Comment:
Is there advantage in still getting all posts in the translation group, as IDs only in line with the other changes in this PR, in one DB operation?